src/algokit/core/typed_client_generation.py

```diff
@@ -27,8 +27,8 @@ class ClientGenerator(abc.ABC):
     language: ClassVar[str]
     extension: ClassVar[str]
 
-    _by_language: dict[str, type["ClientGenerator"]] = {}
-    _by_extension: dict[str, type["ClientGenerator"]] = {}
+    _by_language: dict[str, type["ClientGenerator"]] = {}  # noqa: RUF012
+    _by_extension: dict[str, type["ClientGenerator"]] = {}  # noqa: RUF012
```

Comment:
these shouldn't be suppressed, we should add the `ClassVar` annotation like the rule suggests here.

Reply:
Done